physx.js

```diff
@@ -833,12 +832,20 @@ const physxWorker = (() => {
       meshPositionOffset,
       meshQuaternionOffset,
       hitOffset,
-      idOffset,
     );
 
-    return scratchStack.u32[14] ? {
-      objectId: scratchStack.u32[15],
-    } : null;
+    let head = outputBufferOffset / 4;

```

Comment:
Minor, but rather than hardcoded `4`, we can use `Float32Array.BYTES_PER_ELEMENT`.

Reply:
Changed.